flatgeobuf.cpp

```diff
@@ -168,17 +167,24 @@ void readFeature(const FlatGeobuf::Feature *feature, long long feature_sequence_
 			// https://github.com/protomaps/tippecanoe/issues/7
 			// check if column name is tippecanoe:minzoom, tippecanoe:maxzoom or tippecanoe:layer
 
-			FlatGeobuf::ColumnType col_type = h_column_types[col_idx];
+			FlatGeobuf::ColumnType col_type;
+			if (feature->columns() != NULL) {
```

Comment:
How are you testing this? Last I checked the ogr2ogr writer always puts column type defs in the header.